fairscale/optim/oss.py

```diff
@@ -552,8 +545,11 @@ def _broadcast_params(self) -> None:
 
         for device in self.buckets.keys():
             for src_rank, bucket in enumerate(self.buckets[device]):
-                global_src_rank = self.get_global_rank(self.group, src_rank)
-                last_work_handle = dist.broadcast(tensor=bucket, src=global_src_rank, group=self.group, async_op=True)
+                if bucket.numel() > 0:
```

Comment:
not sure about broadcasting something empty for all backends, and does not make a ton of sense, so just skip that